DROD/EditRoomScreen.cpp

```diff
@@ -7170,6 +7192,13 @@ void CEditRoomScreen::SetSignTextToCurrentRoom(
 	case ES_GETSQUARE:
 		wstrSignText += g_pTheDB->GetMessageText(MID_GetRoomSquare);
 		break;
+	case ES_GETSQUARES1:
+	case ES_GETSQUARES2:
+		wstrSignText += g_pTheDB->GetMessageText(MID_GetRoomSquare);
+		wstrSignText += L" (";
```

Comment:
I think we need to use vars/chars defined in wchar.* for these unicode symbols so this works on other architectures/OSes.